mlflow/pytorch/__init__.py

```diff
@@ -1133,7 +1133,7 @@ def print_auto_logged_info(r):
     )
 
 
-def load_checkpoint(model_class, run_id=None, epoch=None, global_step=None):
+def load_checkpoint(model_class, run_id=None, epoch=None, global_step=None, **kwargs):
```

Comment:
```suggestion
def load_checkpoint(model_class, run_id=None, epoch=None, global_step=None, kwargs):
```

Can we use `kwargs` instead of `**kwargs` to avoid collisions between the `load_checkpoint` arguments and the model class arguments?

Reply:
LGTM. But the last argument `kwargs` needs default value `None`.